index.bs

```diff
@@ -1746,14 +1746,15 @@ during credential generation.
     *   <dfn>none</dfn> - indicates that the [=[RP]=] is not interested in [=authenticator=] [=attestation=].
         The client MAY replace the [=AAGUID=] and [=attestation statement=] generated
         by the authenticator with meaningless client-generated values. For example, in order to avoid having to obtain
-        [=user consent=] to relay uniquely identifying information to the [=[RP]=], or to save a roundtrip to a Privacy CA.
+        [=user consent=] to relay uniquely identifying information to the [=[RP]=], or to save a roundtrip to a
```

Comment:
Change "a" to "an"

Reply:
fixed, see https://github.com/w3c/webauthn/pull/762/commits/0cbccd00839e874b81409414256c6071e322f6aa